File: application/src/main/java/top/emilejones/hhu/application/dto/retrieval/TextType.java
package top.emilejones.hhu.application.dto.retrieval;

public enum TextType {
    COMMON_TEXT("普通文本"),
    TABLE("表格文本"),
    IMAGE("图片文本"),
    TITLE("标题文本");

    private final String comment;

    TextType(String comment) {
        this.comment = comment;
    }

    public String getComment() {
        return comment;
    }
}

File: domain/src/main/kotlin/top/emilejones/hhu/domain/structuresplitter/TextType.kt
package top.emilejones.hhu.domain.structuresplitter

enum class TextType(val comment: String) {
    COMMON_TEXT("普通文本"),
    TABLE("表格文本"),
    IMAGE("图片文本"),
    TITLE("标题文本"),
    NULL("无效节点")
}
